tabbycat/breakqual/base.py

```diff
@@ -255,9 +265,18 @@ def compute_break(self):
         self.breaking_teams = self.eligible_teams[:self.break_size]
 
         # If the last spot is tied, add all tied teams
+        tied_teams_count = 0
         if len(self.eligible_teams) >= self.break_size:
             last_rank = self.eligible_teams[self.break_size-1].get_ranking("rank")
             for tsi in self.eligible_teams[self.break_size:]:
                 if tsi.get_ranking("rank") != last_rank:
                     break
                 self.breaking_teams.append(tsi)
+                tied_teams_count += 1
+
+        # Then add the reserve teams (these are later marked as "reserve" in the
+        # populate database section)
```

Comment:
The logic here should be to include reserve teams up to `len(breaking_teams) == break_size + reserve_size`. Losers in a tie would be first to be a reserve, so adding more wouldn't be necessary.